<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多模态生物特征识别软件v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="SelectWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>250</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="selectlabel">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>21</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>  模式选择</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_face">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>人脸识别</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_fp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>指纹识别</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_pixel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>像素融合</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_iris">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>虹膜识别</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_feature">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>特征融合</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_score">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>分数融合</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_optimal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>最优融合</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ResultWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>250</width>
      <height>300</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>230</width>
       <height>280</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TestWidget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>660</width>
      <height>280</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>660</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>21</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>测试数据</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_face">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_iris">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_fp">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtond_face">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
border-width: 2px;
</string>
     </property>
     <property name="text">
      <string>选择图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtond_iris">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>220</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
border-width: 2px;
</string>
     </property>
     <property name="text">
      <string>选择图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtond_fp">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>220</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
border-width: 2px;
</string>
     </property>
     <property name="text">
      <string>选择图片</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="GalleryWidget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>660</width>
      <height>220</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>660</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>21</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>匹配结果</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_face_g">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_iris_g">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_fp_g">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>960</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>基于深度学习算法的多模态生物特征识别系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="FunctionWidget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>640</y>
      <width>660</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButtond_gallery">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>46</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
border-width: 2px;
</string>
     </property>
     <property name="text">
      <string>选择底库</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>46</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前底库:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="galleryTextBrowser">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>46</y>
       <width>460</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtond_up">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
border-width: 2px;
</string>
     </property>
     <property name="text">
      <string>上一个</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtond_down">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
border-width: 2px;
</string>
     </property>
     <property name="text">
      <string>下一个</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>15</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>展示前</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>个匹配结果</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>12</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtond_res">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
border-width: 2px;
</string>
     </property>
     <property name="text">
      <string>生成结果</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="MyMenu" name="settingMenu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="accelerateAction"/>
    <addaction name="saveAction"/>
   </widget>
   <addaction name="settingMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="accelerateAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>加速模式</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>保存结果</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyMenu</class>
   <extends>QMenu</extends>
   <header location="global">mymenu.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
